### ISSUE TITLE
Chaining MMR and KeyBERTinspired() as respresentation model does not work

### ISSUE THREAD

**Issue by nellieengstrom** [MaartenGr/BERTopic#1224]
Hi Maarten,

I do not understand why I cannot chain MMR and KeyBERTInspired, they work on their own with this code but when combined only KeyBERTInspired is applied, not MMR. For a certain topic, I get keywords that are very similar to each other from only using KeyBERTInspired and would like to use MMR to get more unique words. Any idea why it does not work? I have also tried increasing diversity but nothing changes. 

`keybert_model = KeyBERTInspired(random_state=42)
mmr_model = MaximalMarginalRelevance(diversity=0.7)

representation_models = [mmr_model, keybert_model]

embedding_model = SentenceTransformer('paraphrase-multilingual-MiniLM-L12-v2')
embeddings = embedding_model.encode(data, show_progress_bar=False)

umap_model = UMAP(n_neighbors=15, n_components=5, min_dist=0.0, metric='cosine', random_state=42, low_memory=False)

topic_model = BERTopic(representation_model=representation_models, embedding_model=embedding_model, umap_model=umap_model, verbose=True)
topics, probs = topic_model.fit_transform(data, embeddings)`

Would really appreciate some help,
thank you!

**Comment by MaartenGr** [comment]
That is because you used MMR first and then KeyBERT. Instead, I would advise using KeyBERT first and then using MMR to diversify KeyBERT's output. Also, make sure to set `top_n_words` in KeyBERT to be at least 30 since MMR uses the output of KeyBERT, there needs to be a sufficient number of keywords to diversify.

**Comment by MaartenGr** [comment]
Closing this due to inactivity. Let me know if I need to re-open the issue!
